style.css

```diff
@@ -0,0 +1,117 @@
+header {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 40px;
+  width: 375px;
+}
+
+.msLogo {
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Gill Sans', 'Gill Sans MT', 'Trebuchet MS', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  padding-left: 20px;
+  text-decoration: none;
+}
+
+nav {
+  margin-right: 30px;
+}
+
+.homebody {
+  position: absolute;
+  width: 375px;
+  height: 600px;
+  left: 1px;
+  top: 100px;
+  background: #fff;
+  background-image: url(icon/headerShapes.svg);
+  background-repeat: no-repeat;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 1px 24px;
+  position: absolute;
+  width: 320px;
+  height: 64px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+}
+
+.text1 {
+  position: static;
+  width: 327px;
+  height: 150px;
+  left: 24px;
+  top: 114px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 1px;
+}
+
+#text2 {
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 12px 1px;
+}
+
+.text3 {
+  position: relative;
+  width: 217px;
+  height: 4px;
+  left: 1px;
+  top: 1px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 2px;
+  color: #7f8cff;
+}
+
+.social-icon {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  gap: 12px;
+  width: 270.17px;
+  height: 20px;
+  order: 3;
+  margin: 32px 1px;
+  list-style: none;
+  padding-left: 1px;
+}
```

Comment:
- [ ] Please make the header and the headline section responsive as much as you can, I would suggest making the width 100% so your width is not fixed and can fit different mobile screen sizes.
Your Implementation:
<img width="181" alt="image" src="https://user-images.githubusercontent.com/77190203/149320910-078a73b5-efb7-4fcd-ad57-a4bed5fabeb9.png">